Formula/tarantool.rb

```diff
@@ -39,8 +38,13 @@ def install
     args << "-DENABLE_DIST=ON"
     args << "-DOPENSSL_ROOT_DIR=#{Formula["openssl@1.1"].opt_prefix}"
     args << "-DREADLINE_ROOT=#{Formula["readline"].opt_prefix}"
+    args << "-DENABLE_BUNDLED_LIBCURL=OFF"
     args << "-DCURL_INCLUDE_DIR=#{sdk}/usr/include"
     args << "-DCURL_LIBRARY=/usr/lib/libcurl.dylib"
+    args << "-DCURSES_NEED_NCURSES=TRUE"
+    args << "-DCURSES_NCURSES_INCLUDE_PATH=/usr/include"
+    args << "-DCURSES_NCURSES_LIBRARY=/usr/lib/libncurses.dylib"
+    args << "-DICONV_INCLUDE_DIR=/usr/include"
```

Comment:
I don't know what the library setting is for, but that file doesn't exist on my system

```suggestion
    args << "-DCURSES_NCURSES_INCLUDE_PATH=#{sdk}/usr/include"
    args << "-DCURSES_NCURSES_LIBRARY=/usr/lib/libncurses.dylib"
    args << "-DICONV_INCLUDE_DIR=#{sdk}/usr/include"
```

Reply:
The goal is to get rid of the SDK paths to don't hit to readline.h from it (more details are above). Hm. Whether the build passes on your system after changing the definitions this way?

Reply:
Seeing how Apple is moving away from putting anything in a UNIX standard location like this, I think we should really try and fix the readline issue rather then masking it.

Reply:
Done.

Reply:
Your point is valid too. Sadly, the problem looks deeper than just a broken logic in our cmake scripts. If you're curious, I can try to formulate my thoughts around it. Maybe I miss some widespread approach for libraries search / compiling against them in a complex environment with several library collections of different kinds and origins.

Should cmake ignore libraries in the SDK directories if the SDK is not in the default compiler include paths? If so, why cmake sets -isysroot?